lms/templates/learner_dashboard/program_details_fragment.html

```diff
@@ -9,6 +9,9 @@
 %>
 
 <%block name="js_extra">
+<script type="text/javascript" src="${static.url('common/js/vendor/popper.js')}"></script>
+<script type="text/javascript" src="${static.url('common/js/vendor/bootstrap.js')}"></script>
```

Comment:
Please add a comment noting that this is included here as a temporary fix to an issue with Bootstrap, linking to the ticket to update the way this is implemented.